### PR TITLE
275817 - i18n - domaine resource

### PR DESCRIPTION
**Pull request by MBausson**

### CODE REVIEW

**Review by B-Rass — CHANGES_REQUESTED** [review]
One correction needed, otherwhile good to be merged

File: app/views/domaines/index.html.erb [hunk 1]
```diff
@@ -1,4 +1,7 @@
-<%= render 'layouts/breadcrumb', breadcrumb_variables: {'Paramètres' => modeles_url, 'Domaines' => ''} %>
+<%= render 'layouts/breadcrumb', breadcrumb_variables: {
+  t("breadcrumb.settings") => modeles_url,
```
Comment (nicolas-brousse) [inline app/views/domaines/index.html.erb:2]:
```suggestion
  t("breadcrumb.settings") => domaines_url,
```

File: config/locales/fr.yml [hunk 2]
```diff
@@ -37,7 +40,26 @@ fr:
...
+    new:
+      title: New domaine
+    create:
+      flashes:
+        created: Domaine was successfully created.
+    edit:
+      title: Modifier le domaine
+    update:
+      flashes:
+        updated: Domaine was successfully updated.
```
Comment (nicolas-brousse) [inline config/locales/fr.yml:59]: Translate in French

File: config/locales/fr.yml [hunk 3]
```diff
@@ -37,7 +40,26 @@ fr:
...
+      new_domaine: Ajouter un domaine
+      title: Domaines
+      delete_confirmation: Ce domaine ne pourra pas être supprimé tant qu'il sera associé à au moins un serveur. Voulez-vous continuer ?
+    show:
+      title: Domaine %{name}
+    new:
+      title: New domaine
+    create:
+      flashes:
+        created: Domaine was successfully created.
```
Comment (nicolas-brousse) [inline config/locales/fr.yml:54]: Translate in French

File: config/locales/fr.yml [hunk 4]
```diff
@@ -37,7 +40,26 @@ fr:
...
-
+  domaines:
+    index:
+      new_domaine: Ajouter un domaine
+      title: Domaines
+      delete_confirmation: Ce domaine ne pourra pas être supprimé tant qu'il sera associé à au moins un serveur. Voulez-vous continuer ?
+    show:
+      title: Domaine %{name}
+    new:
+      title: New domaine
```
Comment (nicolas-brousse) [inline config/locales/fr.yml:51]: Translate in French

**Review by nicolas-brousse — CHANGES_REQUESTED** [review]
@MBausson some change to do

File: app/views/domaines/index.html.erb [hunk 5]
```diff
@@ -1,4 +1,7 @@
-<%= render 'layouts/breadcrumb', breadcrumb_variables: {'Paramètres' => modeles_url, 'Domaines' => ''} %>
+<%= render 'layouts/breadcrumb', breadcrumb_variables: {
+  t("breadcrumb.settings") => domaines_url,
+  t("breadcrumb.domaines") => ""}
```
Comment (nicolas-brousse) [inline app/views/domaines/index.html.erb:3]:
```suggestion
  Domaine.model_name.human.pluzalize => ""}
```

I let you clean the locale file too

**Review by nicolas-brousse — CHANGES_REQUESTED** [review]

**Comment by nicolas-brousse** [comment]
@MBausson may you rebase from `master` and resolve conflicts? Otherwise LGTM

**Pull request merged**